tests/functests/CMakeLists.txt

```diff
@@ -15,6 +15,12 @@ if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/modules/privacyguard/ AND BUILD_PRIVACYGUA
   )
 endif()
 
+if(EXISTS ${CMAKE_SOURCE_DIR}/lib/modules/dataviewer/)
```

Comment:
This should work, but also double-check what happens if both main + modules have been checked out. But the build excludes the dataviewer. I think Edge browser does not use your dataviewer implementation, as they have their own that is based on pure debug event callback.

Reply:
hmm, this is the same way we were doing it in UTs though.

Reply:
Okay. Let's see what happens next.

Reply:
May need to double-check with @bliptec on that

Reply:
For better or worse, we maintain our own GN build script for Edge, so the CMakeLists changes don't directly affect us (other than needing to reconcile them against our GN build file when updating)

@jevery23 fyi

Reply:
Awesome. In that case, I'll leave it as it is for now.